docs/UserGuide.md

```diff
@@ -2,41 +2,70 @@
 
 ## Introduction
 
-{Give a product intro}
+`MindMyMoney` (M<sup>3</sup>) was proposed for users with multiple payment methods who wish to track their expenditures. 
+Conducting anecdotal interviews, we realized users face difficulty tracking their expenditures across different payment 
+platforms on vastly different items. `MindMyMoney` was then conceptualized on the basis of tracking user’s expenditures.
 
 ## Quick Start
 
-{Give steps to get started quickly}
-
-1. Ensure that you have Java 11 or above installed.
-1. Down the latest version of `Duke` from [here](http://link.to/duke).
-
-## Features 
-
-{Give detailed description of each feature}
-
-### Adding a todo: `todo`
-Adds a new item to the list of todo items.
-
-Format: `todo n/TODO_NAME d/DEADLINE`
-
-* The `DEADLINE` can be in a natural language format.
-* The `TODO_NAME` cannot contain punctuation.  
-
-Example of usage: 
-
-`todo n/Write the rest of the User Guide d/next week`
-
-`todo n/Refactor the User Guide to remove passive voice d/13/04/2020`
+1. Ensure that you have Java 11 or above installed. 
+2. Down the latest version of `MindMyMoney` from [here](https://github.com/AY2122S2-CS2113T-T10-4/tp).
+
+
+## Commands Summary <br>
+
+| Command | Format, examples                                                            |
+|---------|-----------------------------------------------------------------------------|
+| Help    | `help` <br/> Prints a list of available commands                            |
+| Add     | `add [DESCRIPTION] [AMOUNT]`<br/>e.g `add biscuits 3`                       |
+| List    | `list`<br/> Displays the current list of expenditures stored by the user    |
+| Delete  | `delete [INDEX]`<br/>e.g `delete 1`                                         | 
+| Update  | `update [INDEX] [NEW_DESCRIPTION] [NEW_AMOUNT]`<br/>e.g `update 2 snacks 5` |
+| Exit    | `bye`<br/> Shuts down `MindMyMoney` application                             |
+|
+
+## Features <br>
+The following are features of the `MindMyMoney` application. Please ensure that the format of commands given is accurate<br>
```

Comment:
I think it's good to mention the [SQUARE_BRACKETS] are the compulsory parameters
```suggestion
The following are features of the `MindMyMoney` application. Please ensure that the format of commands given is accurate. Words in `[SQUARE_BRACKETS]` are the parameters. <br>
```